sentences/en/climate_HassClimateGetTemperature.yaml

```
@@ -5,3 +5,4 @@ intents:
       - sentences:
           - "<what_is> [the] <temp>"
           - "<what_is> [the] <temp> in <area>"
+          - "<what_is> [the] <area> <temp>"
```

Hi @waltlillyman, this is definitely not obvious but `<area>` and `<name>` already include an optional `[the]`: https://github.com/home-assistant/intents/blob/e848342fed0328e312ca9efca4e283dd5ee8a546/sentences/en/_common.yaml#L42

---

We will also need to add at least as many test cases as there are input sentences for each intent :+1:

---

Ok, I can resume tomorrow AM, US CT!

---

Thanks! :slightly_smiling_face:

---

Made those corrections and tested locally successfully.

---

This is not solved yet. We should remove `[the]` as it's included in the `<area>` expansion.

```suggestion
          - "<what_is> <area> <temp>"
```